README.md

```diff
@@ -159,32 +158,25 @@ The IBM MQ client on which this library depends is supported on Linux and Window
 ### Configuring your environment
 
 First you must configure your command console environment as described in the [mq-golang Getting Started instructions](https://github.com/ibm-messaging/mq-golang#getting-started) so that the necessary flags are set;
-```
+```bash
 # Configure your Go environment variables (update to match your own setup)
-export GOROOT=/usr/local/go
 export GOPATH=/home/myuser/workspace
-export PATH=$PATH:$GOROOT/bin
+export PATH=$PATH:$GOPATH/bin
```

Comment:
Removing the GOROOT definition assumes that the user has Go installed inside their GOPATH, which in the examples I've seen online (and my own setup) is not always the case, so I would like to retain the separation between GOROOT and GOPATH please.

Reply:
GOROOT is not required now. Now all go tools detect its location internally and automatically.
Go installation implies adding /usr/local/go/bin to PATH (https://golang.org/doc/install#install). So do scripts in deb or rpm packages.
Setting GOPATH and adding $GOPATH/bin to PATH is also should be done during Go installation but it's not mandatory so this could be useful here.

Also I think that people need go library when they have some project in go and want to use IBM messaging. So they should be familiar with go. If not some links should be enough. But it's just my opinion and not my repo.